tidb-server/main.go

```diff
@@ -783,6 +784,15 @@ func setupTracing() {
 	opentracing.SetGlobalTracer(tracer)
 }
 
+func setupGCTuner() {
+	limit, err := memory.MemTotal()
+	if err != nil {
```

Comment:
Is it by design to exit tidb if tidb can't get memTotal?

Reply:
it is a design. I think if it cannot get memTotal. it will causes many tasks to be problematic.

Reply:
Do we have an default value for  memTotal to avoid exit tidb?